packages/contracts-bedrock/test/L2/L2ToL2CrossDomainMessenger.t.sol

```diff
@@ -106,11 +106,11 @@ contract L2ToL2CrossDomainMessengerTest is Test {
             _message: _message
         });
 
-        // Check that the SentMessage event was emitted with the correct parameters
+        // Check that the event was emitted with the correct parameters
         Vm.Log[] memory logs = vm.getRecordedLogs();
         assertEq(logs.length, 1);
         assertEq(
-            abi.decode(logs[0].data, (bytes)),
+            logs[0].data,
```

Comment:
There are now cheatcodes to expect anonymous logs, though we'd need to upgrade our foundry and forge-std versions: https://github.com/foundry-rs/forge-std/blob/4d63c978718517fa02d4e330fbe7372dbb06c2f1/src/Vm.sol#L1778-L1800

Reply:
Thank you @lightclient

Reply:
Let the anonymous events flow!